modules/fedimint-ln-client/src/pay.rs

```diff
@@ -170,8 +170,13 @@ impl LightningPayCreatedOutgoingLnContract {
                 }
                 Err(e) => {
                     e.report_if_important();
+
                     debug!(
-                        "Awaiting output outcome failed, retrying in {}s",
```

Comment:
We should prefer structured logging https://docs.rs/tracing/latest/tracing/#recording-fields over string interpolation.